### ISSUE TITLE
Leaderboard and all submissions disappeared over night

### ISSUE THREAD

**Issue by konbick** [codalab/codalab-competitions#2940]
We started our competition last Monday, and all worked great until Friday. First submissions came in on Wednesday and I was able to download the submissions from the teams and see who scored well in the leaderboard. From Friday or Saturday on it felt like there was a rollback(?) of the servers - while the score is still shown in the graph, the leaderboard is completely empty and when I try to download the csv file or the submission files, the zips are empty. There was one additional submission yesterday and I just see the score, no other information of who submitted it and what their submissions are.

Here is the link: https://competitions.codalab.org/competitions/30990#results

Could someone help us out and tell us what went wrong? Can we repair this competition or do we need to move to a new one? 
I am happy to include you as admins, just give me the name.

@Didayolo

**Comment by Didayolo** [comment]
Hi, thank you for reporting this. I'll share this problem with other admins of the platform.

> I am happy to include you as admins, just give me the name.

My Codalab ID is `pavao`

**Comment by konbick** [comment]
Hi @Didayolo thanks for your reply. This seems partly solved, as I was able to see the submissions from the admin view and click on "SHOW" to show them on the leaderboard. Why they disappeared in the first place is still unclear to me. However, we also changed the settings to "If submission beats old score, put submission on leaderboard" = true. I will include you to the competition, in case you want to check it. Thanks
